hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/metrics/TestS3GatewayMetrics.java

```diff
@@ -198,8 +198,8 @@ public void testCreateBucketFailure() throws Exception {
       fail();
 
     } catch (OS3Exception ex) {
```

Comment:
nit: please use assertThrow() here and in others tests of this class. If this ask is dragging migration jira, feel free to create improvement jira to use `assertThrow()` instead of try-catch block.

Reply:
Done.